### DISCUSSION TITLE
PostgreSQL support

### DISCUSSION THREAD

**Discussion by EddyWallace** [SteezCram/extDB3#9] [Ideas]
Is there currently any plan on supporting PostgreSQL in the near future?

**Comment by SteezCram** [comment]
Actually, the project needs a complete refactorization (the project is old and it's the original code from Torndeco) of the code to add any features. I don't have the time and the knowledge now to make it.

**Reply by EddyWallace** [reply]
Alright.
Let me see if I can find enough spare time to contribute to the project.🙂

**Reply by SteezCram** [reply]
It would be awesome, you have a fully configured visual studio project if you want (you only need to changer the some path to compile it)
